src/client/components/MealList.jsx

```diff
@@ -1,33 +1,57 @@
-import React, { useState, useEffect } from "react";
-const mealsUrl = process.env.REACT_APP_MEALS_URL;
+import React, { useContext, useState, useEffect } from "react";
+import SearchMeals from "./SearchMeals";
+import Meal from "./Meal";
+import { MealContext } from "./MealContext";
+import Navbar from "./Navbar";
+import Footer from "./Footer";
+import SortMenu from "./SortMenu";
 
 function MealList() {
-  const [meals, setMeals] = useState([]);
-
+  const { meals } = useContext(MealContext);
+  const [sortedMeals, setSortedMeals] = useState(meals);
+  const [sortOption, setSortOption] = useState("");
   useEffect(() => {
-    try {
-      async function fetchMeals() {
-        const response = await fetch(`${mealsUrl}meals`);
-        const data = await response.json();
-
-        setMeals(data);
+    const sortFunction = (a, b) => {
```

Comment:
Could the sorting function be extracted into a separate one that takes the sorting parameters? 💡